cmd/smith/app/bundle_controller.go

```diff
@@ -256,7 +256,7 @@ func (c *BundleControllerConstructor) resourceInformers(config *ctrl.Config, cct
 		core_v1.SchemeGroupVersion.WithKind("Secret"):                           core_v1inf.NewSecretInformer,
 		core_v1.SchemeGroupVersion.WithKind("ServiceAccount"):                   core_v1inf.NewServiceAccountInformer,
 		apps_v1.SchemeGroupVersion.WithKind("Deployment"):                       apps_v1inf.NewDeploymentInformer,
-		autoscaling_v2b1.SchemeGroupVersion.WithKind("HorizontalPodAutoScaler"): autoscaling_v2b1inf.NewHorizontalPodAutoscalerInformer,
+		autoscaling_v2b1.SchemeGroupVersion.WithKind("HorizontalPodAutoscaler"): autoscaling_v2b1inf.NewHorizontalPodAutoscalerInformer,
```

Comment:
Should this be catched by integration tests?
IIRC they don't run in Travis anymore, but probably on the local machine there is still a way?

Reply:
We don't have integration tests for kube compute yet (that is coming soon). However I had successfully deployed on Playground prior to this change so I'm not entirely sure how that didn't catc hit.

Reply:
You should be able to run Smith's integration tests locally or against a cluster.